pkg/logs/config/config.go

```diff
@@ -162,7 +163,12 @@ func buildHTTPEndpoints() (*Endpoints, error) {
 		additionals[i].UseSSL = main.UseSSL
 	}
 
-	return NewEndpoints(main, additionals, false, true), nil
+	lingerTime, err := lingerTime(coreConfig.Datadog)
```

Comment:
returning an error here is violent, e.g. the agent won't start, do we really want to do this, should we recover gracefully and use the default value instead ?

Reply:
I have no strong opinion on that, maybe fallback on the default value with a log message is better.

Reply:
sounds good, let's do it